dlt/pipeline/__init__.py

```diff
@@ -245,8 +245,8 @@ def run(
     )
 
 # plug default tracking module
-from dlt.pipeline import trace, track
-trace.TRACKING_MODULE = track
+from dlt.pipeline import trace, track, platform
```

Comment:
ok for platform but maybe we should just say opentelemetry?

Reply:
at the moment it is not opentelemetry format at all. i can rename it, but i would just say we switch to opentelemetry when the prototype is out and then also rename this file.